### ISSUE TITLE
Single-file Compare: "Show Whitespace" doesn't work in right pane

### ISSUE THREAD

**Issue by stadlerb** [eclipse/xtext-eclipse#62]
With current Xtext master: 
1. Open any changed DSL model file (e.g. xtend) in a connected EGit repository, right-click and select Compare With -> HEAD revision (or any other item from the Compare With menu with a single-file selection). 
2. Enable "Show Whitespace Characters" if is isn't already turned on.

Then, the whitespace markers will be displayed in the left pane, but not in the right pane.
This does **not** happen when selecting two files and executing Compare With -> Each other.

**Comment by spoenemann** [comment]
Have you verified this is an Xtext problem and not an EGit problem?

**Comment by cdietrich** [comment]
Yes it is a xtext problem (the way we create viewers editors around the merge compare thing)

**Comment by tomaswolf** [comment]
Is this still a problem with >= EGit 5.3.0?

In EGit, there were changes in the way the compare editor inputs are constructed exactly to deal better with showing whitespace characters.

**Comment by tomaswolf** [comment]
If it still is a problem, then [Eclipse bug 543146, comment 4](https://bugs.eclipse.org/bugs/show_bug.cgi?id=543146#c4) might be relevant.

**Comment by cdietrich** [comment]
no the problem is still there. might be due our heavy customizations
